src/ncepbufr/prepbufr_adpsfc_api.py

```diff
@@ -0,0 +1,234 @@
+# (C) Copyright 2023 NOAA/NWS/NCEP/EMC
```

Comment:
Does it matter if these are add/removed vs 'moved'? If they are 'moved', doesn't it retain the git history?

Reply:
usually GitHub indicates something like 'file renamed without changes', but they look like new files to me in the review